photo-helper/src/main/java/io/intrepid/photohelper/ContextDelegate.java

```diff
@@ -0,0 +1,17 @@
+package io.intrepid.photohelper;
+
+import android.content.ContextWrapper;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+
+/**
+ * Created by GuyIntrepid on 6/1/17.
+ */
+
+interface ContextDelegate {
+    void requestPermissions(@NonNull String[] permissions, int requestCode);
+
+    ContextWrapper getContext();
```

Comment:
What's the reason behind returning ContextWrapper instead of just Context?  Also, can you add a @NonNull annotation to the return variable?  If we can't guarantee it is non-null, then this should have a @Nullable annotation.